### ISSUE TITLE
Tests not passing

### ISSUE THREAD

**Issue by kuttim** [entria/entria-fullstack#79]
``` FAIL   web-app  packages/web/src/__tests__/app.test.tsx
  ● Test suite failed to run

    TypeScript diagnostics (customize using `[jest-config].globals.ts-jest.diagnostics` option):
    packages/web/src/UserList.tsx:34:8 - error TS2531: Object is possibly 'null'.

    34       {users.edges.map(({node}) => (
              ~~~~~
    packages/web/src/UserList.tsx:34:26 - error TS2339: Property 'node' does not exist on type '{ readonly node: { readonly id: string; readonly name: string | null; readonly email: string | null; }; } | null'.

    34       {users.edges.map(({node}) => (```

**Comment by sibelius** [comment]
2 ways to fix this

disable typecheck on tests

or use this helper

https://github.com/zth/relay-utils#collectconnectionnodes

to make sure we typesafe

**Comment by kuttim** [comment]
Thanks, that did seems to get rid of the error, however I'm facing another issue: ```Module '"../../../../../../../Users/Mikk/entria-fullstack/node_modules/@types/react-relay"' has no exported member 'Variables'.``` any ideas?

**Comment by sibelius** [comment]
We can update the relay types or send a or fixing then on DefiniteTyped

**Comment by kuttim** [comment]
I would appreciate it.

**Comment by PlayMa256** [comment]
Closed on #80
